File: app/src/main/java/uk/co/sundroid/util/astro/Position.java
package uk.co.sundroid.util.astro;

public class Position {
	
	private long timestamp;
	
	private double azimuth;
	
	private double appElevation;
	
	public long getTimestamp() {
		return timestamp;
	}

	public void setTimestamp(long timestamp) {
		this.timestamp = timestamp;
	}

	public double getAzimuth() {
		return azimuth;
	}

	public void setAzimuth(double azimuth) {
		this.azimuth = azimuth;
	}

	public double getAppElevation() {
		return appElevation;
	}

	public void setAppElevation(double appElevation) {
		this.appElevation = appElevation;
	}

}

File: app/src/main/kotlin/uk/co/sundroid/util/astro/Position.kt
package uk.co.sundroid.util.astro

class Position(val timestamp: Long, val azimuth: Double, val appElevation: Double) {

    var julianDay: Double = 0.0
    var siderealTime: Double = 0.0
    var trueElevation: Double = 0.0
    var topoRA: Double = 0.0
    var topoDec: Double = 0.0
    var topoDistKm: Double = 0.0
    var topoDistEarthRadii: Double = 0.0
    var geoRA: Double = 0.0
    var geoDec: Double = 0.0
    var geoDistKm: Double = 0.0
    var geoDistEarthRadii: Double = 0.0

    var moonAge: Double = 0.0
    var moonPhase: Double = 0.0
    var moonIllumination: Double = 0.0

}
